### PR TITLE
base and dummy attenuator added

### PR DESCRIPTION
**Pull request by MarcusZuber**

### CODE REVIEW

**Comment by MarcusZuber** [comment]
If nobody complains, this can be merged.

File: concert/devices/attenuators/base.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+from concert.base import State, check, AccessorNotImplementedError
+from concert.base import Parameter
+from concert.devices.base import Device
+
+
+class Attenuator(Device):
+    '''Attenuator base class'''
+    state = State(default='standby')
+    filter = Parameter(help='Filter material')
```
Comment (tfarago) [inline concert/devices/attenuators/base.py]: First, `filter` is a keyword in Python, so the name must be different. Second, these are discrete choices, so I would use [Selection](https://github.com/ufo-kit/concert/blob/master/concert/base.py#L388-L406).

File: concert/devices/attenuators/base.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        return self._get_filter_real()
+
+    def _set_filter_real(self, attenuator):
+        raise AccessorNotImplementedError
+
+    def _get_filter_real(self):
+        raise AccessorNotImplementedError
+
+
+class AttenuatorError(Exception):
```
Comment (tfarago) [inline concert/devices/attenuators/base.py]: A simple `pass` would suffice.

**Comment by tfarago** [comment]
Please make sure you also add a nice test.

**Comment by MarcusZuber** [comment]
@tfarago better?

**Comment by tfarago** [comment]
Yep, shall I merge or do you want to add something?

**Comment by MarcusZuber** [comment]
I will just remove on unused import and squash it.

**Comment by MarcusZuber** [comment]
Done

**Comment by MarcusZuber** [comment]
@farago: +Box

**Comment by tfarago** [comment]
Let's wait with this until the actual implementation works.

**Comment by MarcusZuber** [comment]
@tfarago From my side this is ready for merging.

**Pull request merged**